.travis.yml

```diff
@@ -79,6 +79,13 @@ script:
       ./node_modules/.bin/grunt jest:normal
       git checkout -- src/renderers/dom/shared/ReactDOMFeatureFlags.js
       ./node_modules/.bin/gulp react:extract-errors
+
+      ALL_FILES=`find src -name '*.js' | grep -v umd/ | grep -v __tests__ | grep -v __mocks__`
+      COUNT_ALL_FILES=`echo $ALL_FILES | wc -l`
+      COUNT_WITH_FLOW=`grep '@flow' $ALL_FILES | perl -pe 's/:.+//' | wc -l`
+
+      node scripts/facts-tracker/index.js \
+        "flow-files" "$COUNT_WITH_FLOW/$COUNT_ALL_FILES"
```

Comment:
I wrote it in a way that we can only do one commit on the facts branch per commit on master by being able to pass more arguments. But yeah, I can put this one in the flow matrix and the fiber one in the test matrix.

I don't think we should add a facts matrix to get all of them batched together.